src/main.cpp

```diff
@@ -0,0 +1,426 @@
+//  Used HardwareSerial
+//  UART Port   Rx        Tx      Useable
+//  UART0       GPIO3     GPIO1   Yes
+//  UART1       GPIO9     GPIO10  Yes but requires the reassignment of pins
+//  UART2       GPIO16    GPIO17  Yes
+
+#include <Arduino.h>
+#include <HardwareSerial.h>
+#include "BluetoothSerial.h"
+#include <WiFi.h>
+#include <ArduinoOTA.h>
+
+// ########################## DEFINES ##########################
+#define HOVER_SERIAL_BAUD   115200      // [-] Baud rate for HoverSerial (used to communicate with the hoverboard)
+#define SERIAL_BAUD         9600        // [-] Baud rate for built-in Serial (used for the Serial Monitor)
+#define START_FRAME         0xABCD       // [-] Start frme definition for reliable serial communication
+#define TIME_SEND           50        // [ms] Sending time interval
+#define SPEED_MAX_TEST      200         // [-] Maximum speed for testing
+#define SPEED_STEP          10          // [-] Speed step
+#define DEBUG_RX                        // [-] Debug received data. Prints all bytes to serial (comment-out to disable)
+#define LED_BUILTIN 2
+
+#define SSID "KVANT_24"
+#define WifiPass "75367536"
```

Comment:
storing a wifi password on GitHub is a bad idea

Reply:
This is a free Wi-Fi hotspot for testing.